commands/github-post-standup.js

```diff
@@ -1,54 +1,44 @@
-const fs = require("fs");
+const db = require("../database");
+const { Octokit } = require("@octokit/core");
+const { restEndpointMethods } = require("@octokit/plugin-rest-endpoint-methods");
+const MyOctokit = Octokit.plugin(restEndpointMethods);
 
 module.exports = {
     name: "github-post-standup",
     description: "Post a comment on a GitHub discussion",
-    execute(command, message, args) {
+    usage: "github-post-standup <organization-name> <team-name> <discussion-number> <your-comment>",
+    example: "\n -github-post-standup MLH-Fellowship pod-3-1-3 8 \n" +
+        "**What did you achieve in the last 24 hours?**:\n" +
+        " - \n" +
+        "\n" +
+        "**What are your priorities for the next 24 hours?**:\n" +
+        " - \n" +
+        "\n" +
+        "**Blockers**:\n" +
+        " -\n" +
+        "\n" +
+        "**Shoutouts**:\n" +
+        " - @username for x",
```

Comment:
You know what would make this even better...? 😎 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals